File: pair_impair.js
let nombre, reste;
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Veuillez saisir un nombre : ', (answer) => {
    nombre= answer;
    reste = nombre%2;
    if(reste===0){
        console.log("Ce nombre est pair");
    }else{
        console.log("Ce nombre est impair");
    }
    rl.close();
});

File: pair_impair.java
import java.util.Scanner;

public class Principal {

	public static void main(String[] args) {
	//Déclaration d-une variable
	double nombre, reste;
	//Enregistrement des variables
	Scanner clavier = new Scanner(System.in);
	System.out.print("Veillez saisir un nombre: ");
	nombre = clavier.nextDouble();
	reste = nombre%2;
	if (reste==0) {
		System.out.print("Ce nombre est pair");
	}else {
		System.out.print("Ce nombre est impair");
	}
	clavier.close();

	}

}
